integration-tests/src/test/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -6923,6 +6927,236 @@ public void testLoanTransactionOrderAfterReverseReplay() {
         }
     }
 
+    @Test
```

Comment:
These are super bad quality tests and hard to maintain. Can you please migrate it to using the BaseLoanIntegration test at least or implementing this functional test in the cucumber tests? The latter is preferable.

Reply:
Implemented e2e tests